common/persistence/data_blob.go

```diff
@@ -40,7 +40,7 @@ func NewDataBlob(data []byte, encodingTypeStr string) *commonpb.DataBlob {
 	encodingType, ok := enumspb.EncodingType_value[encodingTypeStr]
 	if !ok || (enumspb.EncodingType(encodingType) != enumspb.ENCODING_TYPE_PROTO3 &&
 		enumspb.EncodingType(encodingType) != enumspb.ENCODING_TYPE_JSON) {
-		panic(fmt.Sprintf("Invalid encoding: \"%v\"", encodingTypeStr))
+		panic(fmt.Sprintf("Invalid encoding: %v", encodingTypeStr))
```

Comment:
maybe `%q`?

Reply:
I thought about it but I don't like `%q`. Because everytime we log the error in JSON logger (which is default) it gets only worse.